src/cpp/client/secure_credentials.cc

```diff
@@ -25,7 +25,6 @@
 #include "grpc/impl/codegen/slice.h"
 #include "grpc/slice.h"
 #include "grpcpp/impl/codegen/status_code_enum.h"
```

Comment:
I think the three includes above also should be in angle brackets.

Reply:
That's right. Done.

Reply:
Thanks.
LGTM